### PR TITLE
chore(onboarding): remove parallax

### LINKED ISSUE #19987: Video assets which are iOS only should not end up in Android bundle
# Problem

The video asset files used for onboarding animation are ~ 30 MB and they end up in the Android APK.
The component which uses those assets is not rendered in Android but maybe because its a runtime check those assets make their way into the bundle anyway.

We should find a way to make sure those video assets are included in iOS only so that we can shave off 30 MB from the Android APK.
 
Related links : 
- https://reactnative.dev/docs/images#images-from-hybrid-apps-resources
- https://reactnative.dev/docs/platform-specific-code#platform-specific-extensions

### PR DESCRIPTION
**Pull request by J-Son89**
The decision has been made to remove the parallax and animated videos.

There are many reasons for this, mostly summarised here: https://www.notion.so/3D-Animations-in-mobile-053f629f1a4541a69fb8054b3f384ddb

We will bake the cake first, the cherry can follow afterwards

Props again to @briansztamfater for coming up with this solution at the time! a lot of thought and effort went into it 🙌 

I don't think this needs test as the fallback solution was already in place before. 

Please let me know if I missed anything in the clean up.

fixes: https://github.com/status-im/status-mobile/issues/19987

### CODE REVIEW

File: src/status_im/common/resources.cljs
Comment (J-Son89) [inline src/status_im/common/resources.cljs:131]: deleted all associated videos 👍

**Review by ilmotta — APPROVED** [review]

**Review by shivekkhurana — APPROVED** [review]

File: src/status_im/contexts/wallet/wallet_connect/utils.cljs
Comment (siddarthkay) [inline src/status_im/contexts/wallet/wallet_connect/utils.cljs]: probably this change should be reverted in light of the comment above : 

> ;; NOTE: Not sorting namespaces since @walletconnect/react-native-compat should be the first

**Review by siddarthkay — APPROVED** [review]
NICE
Screenshot 2024-05-15 at 9 51 10 AM

**Comment by siddarthkay** [comment]
Also 2 other things : 
- `make pod-install` should be run so that `Podfile.lock` is updated and removes the reference to `react-native-transparent-video`
-  `make nix-update-gradle` should be run to remove any `gradle` deps added by `react-native-transparent-video`

**Comment by J-Son89** [comment]
> Also 2 other things :
> 
> * `pod install` should be run so that `Podfile.lock` is updated and removes the reference to `react-native-transparent-video`
> * `make nix-update-gradle` should be run to remove any `gradle` deps added by `react-native-transparent-video`

thanks @siddarthkay!

File: src/status_im/contexts/wallet/wallet_connect/utils.cljs
Comment (J-Son89) [reply src/status_im/contexts/wallet/wallet_connect/utils.cljs]: Aha, this is being done by the linter.
cc @ilmotta, @clauxx  @briansztamfater - do we have a way to stop sorting automatically on some files? 🤔

**Comment by J-Son89** [comment]
@siddarthkay - what do you think now? 🤔

File: nix/deps/gradle/proj.list
Comment (siddarthkay) [inline nix/deps/gradle/proj.list]: hmm this is a problem. 
seems like `make nix-update-gradle` command terminated midway..
I can push a commit and fix if you'd like @J-Son89

Comment (J-Son89) [reply nix/deps/gradle/proj.list]: That'd be great! Thanks 🙏

Comment (siddarthkay) [reply nix/deps/gradle/proj.list]: Done.

**Review by jakubgs — APPROVED** [review]

File: src/status_im/contexts/wallet/wallet_connect/utils.cljs
Comment (ilmotta) [reply src/status_im/contexts/wallet/wallet_connect/utils.cljs]: We do @J-Son89, but only if we stop linting the entire file. Check the line calling `clojure-lsp <flags> clean-ns` command in the `lint-fix` target. You can add another file there to be ignored.

**Review by flexsurfer — APPROVED** [review]
finally

**Comment by J-Son89** [comment]
@status-im/mobile-qa - what do you think about skipping QA here, I will do a happy path check myself too 👍

**Comment by qoqobolo** [comment]
> @status-im/mobile-qa - what do you think about skipping QA here, I will do a happy path check myself too 👍

Sounds good, thank you @J-Son89 🙌
If we find any regression later, we will report it separately

**Comment by J-Son89** [comment]
@qoqobolo - this seems to be fine on ios 12 previously whitelisted device - let me know about e2e results and I'll merge when it's all good 👍

**Review by briansztamfater — APPROVED** [review]
Bye bye parallax 👋 😢

File: src/status_im/contexts/wallet/wallet_connect/utils.cljs
Comment (J-Son89) [reply src/status_im/contexts/wallet/wallet_connect/utils.cljs]: @ajayesivan

**Comment by qoqobolo** [comment]
@J-Son89 e2e look good, the failures are not related to PR.
One question about how the illustrations look on iPhone SE. WDYT? Should we adjust their size to fit the smaller screen before merging?

Screenshot 2024-05-15 at 14 28 23

Screenshot 2024-05-15 at 14 28 06

**Comment by siddarthkay** [comment]
We forgot to add `yarn.lock` in this PR...
I'm adding the yarn.lock in my PR here -> https://github.com/status-im/status-mobile/pull/20044

**Pull request merged**
